paper/paper.md

```diff
@@ -0,0 +1,295 @@
+  *safely* at different macroscopic scales the structural and functional systems of the 
+  brain by modeling connections between brain areas as a graph or network, known as
+  `connectome`, and has become a widely used set of methods in Neuroscience [@Bassett2017]. 
+While brain areas are usually derived from high resolution structural T1 weighted MRI (sMRI),
+  structural brain connectomes are mapped from diffusion MR imaging (dMRI) tractography, and
+  functional brain connectomes are usually mapped from resting-state functional MRI (rfMRI).
+As MRI is being increasingly more accessible and used in both clinical and research settings,
+  such multi-modal MRI datasets are being gathered at an unprecedented rate.
+The size and organization of these datasets as well as the increasing complexity of the processing
+  pipelines to analyze them present important challenges for scalable computing, data sharing,
+  reproducibility and transparency of the analysis.
+
+The last ten years have indeed witnessed a number of technical advances and a trend towards the adoption
+  of open research practices such as *open data* and *open methodology*
+  [@Nichols:2017] [@EglenTowardNeuroscience:2017] [@Kennedy2019].
+This has led in particular to the creation of a community standard for dataset organization, known as the Brain Imaging Data Structure (BIDS),
+  initially designed to ease the practice of sharing raw MRI data [@GorgolewskiTheExperiments:2016]. 
+Combined with advances in software virtualization, BIDS has enabled the creation of the BIDS Apps
+  framework which uses software container technology to encapsulate neuroimaging processing pipelines
+  and ensures portability and reproducibility [@GorgolewskiBIDSMethods:2017].
+A large ecosystem of processing pipelines supporting the mapping of connectomes has evolved around this framework,
+  including C-PAC [@cpac:2013], NIAK [@BellecNeuroimagingNIAK:2016], fMRIPrep [@Ghosh:2018], dMRIPrep [@dmriprep:2019],
+  QSIPREP [@CieslakQSIPrep:2020], MRtrix3_connectome [@Smith2019:BIDSApp], NDMG [@KiarAVariability:2018]
+  PyNets [@PisnerPyNets:2020], and Micapipe [@Rodriguez:2022] which have all demonstrated their capability in
+  addressing all the previously-mentioned challenges.
+However, none of the existing solutions provide a direct alternative to Connectome Mapper
+  when dealing with multimodal datasets with the goal to create connectomes at multiple
+  scales with hierarchical region grouping.
+
+Connectome Mapper (CMP), an open-source pipeline software with a graphical user interface,
+  was created before the emergence of BIDS to simplify the organization, processing, and analysis of
+  sMRI, dMRI, and rfMRI from raw data to multi-scale structural weighted and functional
+  connectomes [@Daducci:2012], using in a common framework a multi-scale extension of
+  the Desikan-Killiany parcellation [@Desikan2006AnInterest][@Cammoun2012:MappingMRI].
```

Comment:
Some version of this sentence should imo be at the very beginning of the paper. Right now, people read two full paragraphs before cmp is mentioned.